### ISSUE TITLE
Aria-live should be more assertive in how to cite component

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#5193]
Currently, the how to cite component uses `aria-live="polite"` to announce when the citation has been changed. The screen reader testers suggested this information was not getting through, and the accessibility auditor suggested we change this to `aria-live="assertive"`.

**Comment by asmecher** [comment]
Merged to `main`.
